client/my-sites/email/email-management/page.jsx

```diff
@@ -0,0 +1,18 @@
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import EmailManagementHome from 'calypso/my-sites/email/email-management/email-home';
+import { emailManagement } from 'calypso/my-sites/email/paths';
+
+const EmailManagementHomePage = ( { source, selectedDomainName } ) => {
+	return (
+		<>
+			<PageViewTracker
+				path={ emailManagement( ':site', selectedDomainName ? ':domain' : null ) }
```

Comment:
Why do we pass the domain name here, we don't do that anywhere else when calling `<PageViewTracker/>`?

Reply:
I'm not actually passing the domain itself, more like the domain path marker.
The path could be different based on if the selectDomainName is set.

Reply:
Good catch - this should always be `:domain`.

Reply:
Actually, you are right, 'Seni. This can be `:domain` or null.

Reply:
First, why would `selectedDomainName` be null? If we don't specify a domain name in the url (i.e. https://wordpress.com/email) then we are presented with the site picker, not this page. Second, we don't perform that check anywhere else: why do we want to have a specific behavior for that particular page?

Reply:
There is another case where we show this page: `/email/:siteSlug`
For that case, the domain isn't specified, but we do have the site.

Reply:
Ah right, my bad. I'm not sure what I was thinking this morning 🤦‍♂️.